README.md

```diff
@@ -1 +1,48 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+This is the first project in the APIs week at Microverse. It is a basic leaderboard with the use of custom Api from Microverse capstone Project
+
+![screenshot](./lead.png)
+
+## Built With
+
+- HTML5
+- CSS
+- Javascript
+- Webpack
+- npm
+
+## Live Demo
+
+[Live Demo Link](https://delhinrharl.github.io/Leaderboard/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+## Install
+
+In your terminal, navigate to your current directory and run this code
+
+`git clone https://github.com/DelhinRharl/Leaderboard.git`
+
+Install npm or if installed already using this lin
+`npm install`
```

Comment:
- [ ] After cloning the repo, I think the next step would be `cd Leaderboard`. Otherwise `npm i` and `npm start` not working,